security/fma-u16a.md

```diff
@@ -0,0 +1,171 @@
+
+### Pause mechanism identifier change
+
+Chains without ETHLockbox will use the `OptimismPortal` address as their identifier instead of the lockbox address. Incident response runbooks will be updated accordingly.
+
+## Failure Modes
+
+### FM1: Misconfigured Feature Toggles
+
+#### Description
+
+Chains may incorrectly configure the system-level feature toggles introduced in U16a. This includes enabling ETHLockbox flag without having the ETHLockbox contract deployed, failing to enable ETHLockbox when it's needed, or setting arbitrary unsupported feature flags.
+
+#### Risk Assessment
+
+- Impact: LOW/MEDIUM
+  - Reasoning: Most misconfigurations have no functional effect or cause temporary bridge issues that can be resolved by admin intervention. The most impactful scenario is failing to enable ETHLockbox when needed, which temporarily breaks the bridge.
+- Likelihood: LOW/MEDIUM
+  - Reasoning: Feature toggle system is new, increasing chance of operator error. However, validator checks and clear documentation should minimize risks.
+
+#### Mitigations
+
+- ETHLockbox can only be enabled if lockbox address is set (requires U16 upgrade path).
+- Admin can set toggle post-upgrade if initially misconfigured.
+- Clear operational documentation and runbooks.
+
+#### Detection
+
+- Monitor for portal/lockbox balances.
+- Monitor for unexpected feature flag changes.
+
+#### Action Items
+
+- [ ] Update standard validator to verify lockbox/feature flag configuartion
+- [ ] Set up system feature flag monitor
+- [ ] Set up portal/lockbox balance monitor
+
+#### Recovery Path(s)
+
+- Correct misconfigured toggles via admin transactions.
+- No special recovery procedures beyond standard admin intervention.
+
+### FM2: ETHLockbox Migration Gap
+
+#### Description
+
+Chains upgrading from U15 directly to U16a cannot adopt ETHLockbox until a later upgrade. This creates operational complexity as some chains have ETHLockbox while others don't, leading to testing asymmetry and incident response divergence in pause mechanism runbooks.
+
+#### Risk Assessment
+
+- Impact: LOW
+  - Reasoning: Operational overhead is minimal and doesn't affect core functionality or safety. Primarily creates complexity in testing and operational procedures.
+- Likelihood: HIGH
+  - Reasoning: This is an intentional design decision that will definitely occur for chains following the U15 => U16a upgrade path.
+
+#### Mitigations
+
+- Testing covers both scenarios since major chains exist in both categories.
+- Runbooks and supporting documentation will be updated to prevent operator confusion.
+- Clear documentation of which chains have ETHLockbox and which don't.
+
+#### Detection
+
+- Track which chains follow which upgrade path.
+- Monitor for operator confusion in incident response procedures.
+
+#### Action Items
+
+- [ ] Update incident response runbooks to handle both ETHLockbox and non-ETHLockbox chains
```

Comment:
We also need to (and are already in the process of) updating pause tooling to handle this difference as well